### PR TITLE
fix(npm-audit): improve audit and fix run-scripts

### PR DESCRIPTION
**Pull request by raisedadead**
This applies and audit fix, and uses `--stream` flag to improve logging when using `lerna exec`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
   "main": "none",
   "scripts": {
     "analyze-bundle": "webpack-bundle-analyzer",
-    "audit": "npm audit & lerna exec --no-bail 'echo $LERNA_PACKAGE_NAME & npm audit'",
-    "audit:fix": "npm audit fix & lerna exec 'echo $LERNA_PACKAGE_NAME & npm audit fix'",
+    "audit": "npm audit & npx lerna exec --concurrency 1 --stream -- npm audit",
```
Comment (raisedadead) [inline package.json:23]:
```suggestion
    "audit": "npm audit & npx lerna exec --no-bail --concurrency 1 --stream -- npm audit",
```

File: package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
...
+    "audit:fix": "npm audit fix & npx lerna exec --concurrency 1 --stream -- npm audit fix",
```
Comment (raisedadead) [inline package.json:24]:
```suggestion
    "audit:fix": "npm audit fix & npx lerna exec --no-bail --concurrency 1 --stream -- npm audit fix",
```

**Review by raisedadead — COMMENTED** [review]
Missed adding the `no-bail` flag.

**Review by ojeytonwilliams — APPROVED** [review]
This is so much easier to read than the old version. LGTM :+1: 

Weirdly, `--no-bail` didn't seem necessary for the `fix` version, but I don't see the harm in using it.

**Pull request merged**